EIPS/eip-7331.md

```diff
@@ -0,0 +1,403 @@
+---
+eip: 7331
+title: Private Equity
+description: A private equity fund Contract distributes dividends of a private equity fund in stable coins or in fund tokens as per the user's choice.
+author: Oasis Pro Markets (@tokenizeassets)
+discussions-to: https://ethereum-magicians.org/t/a-proposal-for-private-equity-and-funds-smart-contract/14681
+status: Draft
+type: Standards Track
+category: ERC
+created: 2023-07-04
+requires: 20, 3643
+---
+
+## Abstract
+
+A private equity fund is a type of investment fund that pools together capital from high-net-worth individuals, institutional investors, and sometimes, pension funds to make investments in private companies. These funds are managed by private equity firms, which specialize in acquiring ownership stakes in companies and actively managing them to generate returns.
```

Comment:
This doesn't really explain what this ERC does.

Reply:
I would say more along the lines of "this standard proposes a standardized interface for Private Equity smart contracts" or something